ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -132,6 +132,74 @@
     },
 }
 
+CH2PP_TXN_QUERIES = {
+    "DELIVERY": {
+        "beginWork": "BEGIN WORK",
+        "rollbackWork":"ROLLBACK WORK",
+        "commitWork":"COMMIT WORK",
+        "getNewOrder": "SELECT no_o_id FROM default:bench.ch2.neworder WHERE no_d_id = $1 AND no_w_id = $2 AND no_o_id > -1 LIMIT 1", #
+        "deleteNewOrder": "DELETE FROM default:bench.ch2.neworder WHERE no_d_id = $1 AND no_w_id = $2 AND no_o_id = $3", # d_id, w_id, no_o_id
+        "getCId": "SELECT o_c_id FROM default:bench.ch2.orders WHERE o_id = $1 AND o_d_id = $2 AND o_w_id = $3", # no_o_id, d_id, w_id
+        "updateOrders": "UPDATE default:bench.ch2.orders SET o_carrier_id = $1 WHERE o_id = $2 AND o_d_id = $3 AND o_w_id = $4", # o_carrier_id, no_o_id, d_id, w_id
+#        "updateOrderLine": "UPDATE default:bench.ch2.ORDER_LINE SET OL_DELIVERY_D = $1 WHERE OL_O_ID = $2 AND OL_D_ID = $3 AND OL_W_ID = $4", # o_entry_d, no_o_id, d_id, w_id
```

Comment:
Is there a reason to keep all these commented out lines? Also, do we need the comments at the end of the lines? Are they supposed to indicate which columns are being accessed?